internal/authprovider/ldap.go

```diff
@@ -301,6 +322,21 @@ func newUserApplicationSearchRequest(userDN string) *ldap.SearchRequest {
 	}
 }
 
+// request to get user group list
+func newUserViewerSearchRequest(userDN string) *ldap.SearchRequest {
```

Comment:
does that really need to be a function, as it just returns a SearchRequest, isn't tested, and really is not reused in multiple places?